src/base/net/geoipmanager.cpp

```diff
@@ -43,8 +44,8 @@
 #include "geoipdatabase.h"
 
 const QString DATABASE_URL = QStringLiteral("https://download.db-ip.com/free/dbip-country-lite-%1.mmdb.gz");
-const char GEODB_FOLDER[] = "GeoDB";
-const char GEODB_FILENAME[] = "dbip-country-lite.mmdb";
+const Path GEODB_FOLDER {u"GeoDB"_qs};
+const Path GEODB_FILENAME {u"dbip-country-lite.mmdb"_qs};
```

Comment:
You insisted before on treating file name as string and not as path.

Reply:
Sorry, reverted.